tests/testthat/test-change.R

```diff
@@ -32,13 +32,13 @@ testthat::test_that("arc works", {
 
   # vector
   testthat::expect_equivalent(
-    arc(years, values), c(NA, 46.05, 32.19), tolerance = 0.02
+    arc(years, values), c(NA, 13.86, 4.46), tolerance = 0.02
```

Comment:
Is this using an example from the UN link cited? Could use the example from preston pg 17. `N(10) = 134,986` `N(0) = 100,000` `r = 0.03`